device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C28S4/BALANCED/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,52 @@
+{%- set default_cable = '300m' %}
```

Comment:
this PR looks the same https://github.com/sonic-net/sonic-buildimage/pull/22117/files. Not sure if I missed anything.

Reply:
i was not aware of the other change. Then this one could be closed. Thanks.